tracing/README.md

```diff
@@ -0,0 +1,37 @@
+# Tracing package
+
+`package tracing` provides an API wrapper around the tracing library
+[jaeger](https://github.com/uber/jaeger-client-go) that allows you to
+instrument operations in your application.
+Jaeger tracer can be configured with an optional logger that logs errors/spans
+and a stats reporter for emitting metrics.
+Note that the tracer is initialized by default with usage of UberFx modules.
```

Comment:
You could also make this more straightforward for users—something like:
"Using UberFx modules initializes Jaeger tracing by default."